.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Unit Tests
+
+# Remove this for now, circleci is getting in the way with tests failing
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  # trigger on pull requests
+  pull_request:
+
+  # trigger on all commits to master and tagged commits
```

Comment:
I'd like to see the project rename `master` ->` main` in alignment with a large portion of the Python ecosystem and open source community. Maybe this is as good a time as any.

Reply:
I've never understood to motivation for such a change, personally. If that change would make the package more professional looking, though, I can support it. For the sake of PR modularity, it may be best suited for another PR, but I'm still on the fence about renaming it.

Reply:
These resources help explain why projects are making this move. I'm happy with keeping PRs modular, it doesn't have to be done now but should be considered in the near future. signac plans to do the same migration, probably around the time of the 2.0 release.
https://github.com/github/renaming
https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main
https://www.zdnet.com/article/github-to-replace-master-with-main-starting-next-month/